### ISSUE TITLE
Confused atlas loading in vue

### ISSUE THREAD

**Issue by ODtian** [pixijs/pixi-spine#372]
I'm trying to use pixi-spine (both latest and 2.0.5, and my spine is 3.6) with vue, but when I try to load atlas file, I got my spineData missed, I've check my network and request looks well, however I just can't see spineData anywhere, even the string is loaded at data property, I will post code and error in console below.

```
<script>
import * as PIXI from "pixi.js";

window.PIXI = PIXI;
require("pixi-spine");

export default {
    name: "SpinePlayer",
    props: {
        atlasUrl: { type: String, default: "/spine/100361/100361.skel" },
    },
    data() {
        return {
            pixiApp: null,
            spine: null,
        };
    },
    mounted() {
        this.addApp();
        this.addSpine();
    },
    methods: {
        addApp() {
            this.pixiApp = new PIXI.Application({
                transparent: true,
                antialias: true,
                resolution: 1,
                backgroundColor: 0,
            });
        },
        addSpine() {
            const name = "spineCharacter" + Math.random();
            this.pixiApp.loader.add(name, this.atlasUrl).load((loader, resources) => {
                console.log(resources[name]);
                if (this.spine == null) {
                    this.spine = new PIXI.spine.Spine(resources[name].spineData);
                    this.pixiApp.stage.addChild(this.spine);
                } else {
                    this.spine = new PIXI.spine.Spine(resources[name].spineData);
                }
            });
        },
    },
};
</script>
```
![image](https://user-images.githubusercontent.com/40318251/106234061-ec766100-6232-11eb-8b2b-5ebcfd137c01.png)
![image](https://user-images.githubusercontent.com/40318251/106234110-09129900-6233-11eb-9dee-e6b45df89224.png)

looks fine
![image](https://user-images.githubusercontent.com/40318251/106234195-470fbd00-6233-11eb-82ae-5cd16bcc5e2c.png)
![image](https://user-images.githubusercontent.com/40318251/106234217-4e36cb00-6233-11eb-8db1-3d374eccc349.png)

**Comment by ODtian** [comment]
might there are some issues with my skel file because i found that i cant use any tool convert it to json

**Comment by ivanpopelyshev** [comment]
Write here if it still doesnt work, i can point you where to place breakpoints

**Comment by ODtian** [comment]
Yes i really need some help. I'm trying play the spine downloaded from this repo:AnimPlayer, it provides skel and png atlas downloading(online version https://redive.estertion.win/spine/). The player is running on spine-ts/webgl which worked fine. I download spine-ts and do all setup manually, I have compared all the thing and found that it always failed at reading the data from a binary array to get the skeletonData, the most weird things is that i placed breakpoints everywhere and didn't find anything different until my player raise an error, I will post that below

**Comment by ivanpopelyshev** [comment]
oh, interesting, you can send it to ivan.popelyshev at gmail.com if you dont want to paste here, i'll help ) I love it when people know how to debug

**Comment by AvetisSargsian** [comment]
Hello,
I have the same issue with loading *.skel file, Loader defines the type of file incorrectly.
I would appreciate some help, please.

"pixi-spine": "^3.0.13",
 "pixi.js": "^6.2.2",


<img width="430" alt="Screenshot 2022-04-18 at 14 58 11" src="https://user-images.githubusercontent.com/8079621/163805476-d1a2b99d-dd49-4c52-bf2f-946bafc3355a.png">
